raven/backend.go

```diff
@@ -28,17 +28,56 @@ func init() {
 // sent to sentry they are tagged as coming from the given
 // project. It then sets up the connection to sentry and begins
 // to send any errors recieved over comm to sentry.
-// It panics if a client could be initialized.
+// It panics if a client could not be initialized.
 func CaptureErrors(project, dsn string, comm <-chan glog.Event) {
 	projectName = project
 	client, err := NewClient(dsn)
 	if err != nil {
 		panic(err)
 	}
 
-	for e := range comm {
-		if e.Severity == "ERROR" {
-			client.CaptureGlogEvent(e)
+	for glogEve := range comm {
+		if glogEve.Severity == "ERROR" {
+			client.CaptureGlogEvent(glogEve)
+		}
+	}
+}
+
+// CaptureErrorsDsnOptions allows you to have errors sent to one of multiple dsn targes.
+// It sets up a connection to sentry for each of the given dsn URIs.
+// It panics if any client could not be initialized.
+// The target dsn of each event is determined by the "dsn" field in the event's
+// data map. If an event has no such field or the field is not equal to
+// any of the dsns initially passed in, the dsn target will be assumed
+// to be the first dsn in the dsns list.
+func CaptureErrorsDsnOptions(project string, dsns []string, comm <-chan glog.Event) {
```

Comment:
I didn't want to use Multi because that sounds like the event will go to more than one. Option implies only one.